example/src/Examples/Drawing/Context/functions/findClosestElementToPoint.ts

```diff
@@ -1,11 +1,11 @@
-import type { Point } from "@shopify/react-native-skia";
+import type { IPoint } from "@shopify/react-native-skia";
```

Comment:
I think we asked this before as well - why is this changed from `Point` -> `IPoint`?